packages/core/src/components/hero/hero.css

```diff
@@ -278,6 +278,10 @@
     padding-right: var(--spacing-m);
     padding-top: var(--spacing-xs);
   }
+
+  .hds-hero__title {
+    font-size: var(--fontsize-heading-xl);
```

Comment:
Specifications were not the clearest. Use --fontsize-heading-xl-mobile also in this breakpoint between 768px and 992px.